### ISSUE TITLE
eleos - multi-platform graphical wallet for zero-knowledge cryptocurrencies

### ISSUE THREAD

**Issue by diehard2k9** [zencashio/eleos#9]
![eleos wallet error](https://cloud.githubusercontent.com/assets/26531346/25159631/14a6d790-247f-11e7-9da0-70823794e05b.jpg)
I get this error when clicking address.
the Zcash wallet works fine, I didnt test it much
zcl-cli getinfo was up today

**Comment by joshuayabut** [comment]
Try installing v0.0.4 version and let me know if you still get this error.

**Comment by sparker327** [comment]
I get the same error on Win 10, ver v0.0.4. At first everything loaded, even though I received javascript error upon first opening of program. Then it only showed two currencies, Zen not one of them. I found the updated files, imported them into the folders and upon opening again, I saw Zen. When I selected zen, closed the wallet and re-opened, it kept giving me javascript error and wallet is unable to open.
